### ISSUE TITLE
Style header links to indicate active page

### ISSUE THREAD

**Issue by eburdekin** [hackforla/expunge-assist#1420]
### Overview
Adding styling to our header links to indicate the active page would help users keep track of which section of the site they're on among Home, About Us, or FAQ.

Note for Sydney: creating this as an `issue level: request` for now, can update to a `main` with 2 dependencies later if the team wants to move forward with making this update?

### Action Items
- [ ] Design team to determine styling for active page's header link (maybe same as hover state?)
- [ ] Dev team to code header styling update

**Comment by anitadesigns** [comment]
@sydneywalcoff Agreed that this is a good to implement! I went ahead and making this issue the "Main" issue

**Comment by sydneywalcoff** [comment]
@anitadesigns perfect! Thank you!
